signify-ts-test/test-multiple-aids.sh

```diff
@@ -0,0 +1,99 @@
+
+# Print environment variable values
+echo "TEST_ENVIRONMENT=$TEST_ENVIRONMENT"
+echo "ROLE_NAME=$ROLE_NAME"
+echo "REG_PILOT_API=$REG_PILOT_API"
+echo "VLEI_VERIFIER=$VLEI_VERIFIER"
+echo "KERIA=$KERIA"
+echo "KERIA_BOOT=$KERIA_BOOT"
+echo "WITNESS_URLS=$WITNESS_URLS"
+echo "WITNESS_IDS=$WITNESS_IDS"
+echo "VLEI_SERVER=$VLEI_SERVER"
+
+# Check if the only argument is --all
+if [[ $# -eq 1 && $1 == "--all" ]]; then
+    set -- --docker=verify --build --data --report --verify
+fi
+
+# Parse arguments
+while [[ $# -gt 0 ]]; do
+    case $1 in
+        --docker=*)
+            docker_action="${1#*=}"
+            case $docker_action in
+                deps | verify)
+                    docker compose down -v
+                    docker compose up $docker_action -d
+                    ;;
+                *)
+                    echo "Unknown docker action: $docker_action"
+                    ;;
+            esac
+            shift # past argument
+            ;;
+        --build)
+            npm run build
+            shift # past argument
+            ;;
+        --data)
+            export LEI=875500ELOZEL05BVXV37
+            export SIGNIFY_SECRETS=D_PbQb0ESFGQgK-kDWd3f,BTaq43FG3G34GGSQJp6mb,BRFHnuf98vnih8hfinf4g,nf98hUHUy8Vt5tvdyaYV7
+            npx jest ./singlesig-vlei-issuance.test.ts
+            shift # past argument
+            export LEI=875500ELOZEL05BVXV37
+            export SIGNIFY_SECRETS=D_PbQb01zuzQgK-kDWjqy,BTaqgh1eeOjXO5iQJp6mb,Akv4TFoiYeHNqzj3N8gEg,CbII3tno87wn3uGBP12qm
+            npx jest ./singlesig-vlei-issuance.test.ts
+            shift # past argument
+            export LEI=875500ELOZNFID93NF83
+            export SIGNIFY_SECRETS=D_PbQfw4fvFEf4-fesefF,NUH98hbsuhd8hdua8hnfa,ANud8hnahdaNUnfsanofN,defh7b7g7gfaBuf83bf0f
+            npx jest ./singlesig-vlei-issuance.test.ts
+            shift # past argument
+            ;;
+        --report)
+            export LEI=875500ELOZEL05BVXV37
+            export SIGNIFY_SECRETS=D_PbQb0ESFGQgK-kDWd3f,BTaq43FG3G34GGSQJp6mb,BRFHnuf98vnih8hfinf4g,nf98hUHUy8Vt5tvdyaYV7
+            npx jest ./report.test.ts
+            shift # past argument
+            export LEI=875500ELOZEL05BVXV37
+            export SIGNIFY_SECRETS=D_PbQb01zuzQgK-kDWjqy,BTaqgh1eeOjXO5iQJp6mb,Akv4TFoiYeHNqzj3N8gEg,CbII3tno87wn3uGBP12qm
+            npx jest ./report.test.ts
+            shift # past argument
+            export LEI=875500ELOZNFID93NF83
+            export SIGNIFY_SECRETS=D_PbQfw4fvFEf4-fesefF,NUH98hbsuhd8hdua8hnfa,ANud8hnahdaNUnfsanofN,defh7b7g7gfaBuf83bf0f
+            npx jest ./report.test.ts
+            shift # past argument
+            ;;     
+        --verify)
+            export SIGNIFY_SECRETS_MULTI_AID="D_PbQb0ESFGQgK-kDWd3f,BTaq43FG3G34GGSQJp6mb,BRFHnuf98vnih8hfinf4g,nf98hUHUy8Vt5tvdyaYV7;D_PbQb01zuzQgK-kDWjqy,BTaqgh1eeOjXO5iQJp6mb,Akv4TFoiYeHNqzj3N8gEg,CbII3tno87wn3uGBP12qm;D_PbQfw4fvFEf4-fesefF,NUH98hbsuhd8hdua8hnfa,ANud8hnahdaNUnfsanofN,defh7b7g7gfaBuf83bf0f"
```

Comment:
Remind me, the third secret, for the first two secrets sets, should be the same? I think that third secret is the LE secret and that LE should have issued the role credential to each of the AIDs (protected by the fourth secret)?